modules/reindex/src/main/java/org/elasticsearch/index/reindex/RemoteScrollableHitSource.java

```diff
@@ -150,25 +147,48 @@ HttpPost buildStartRequest() {
                 }
             }
             if (useScan) {
-                uri.append("&search_type=scan");
+                uri.addParameter("search_type", "scan");
```

Comment:
> scan search_type was already depricated in 2.1. Isn't it tto be removed in 5.0?

This is reindex from remote, which means that it needs to support the API of previous versions of Elasticsearch.